integrations/database_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"fmt"
+	"testing"
+
+	"code.gitea.io/gitea/models/db"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDatabase(t *testing.T) {
+	defer prepareTestEnv(t)()
+
+	// test a SQL string
+	assert.Equal(t, "'it''s fine'", db.QuoteString(db.DefaultContext, "it's fine"))
+
+	// test all ASCII chars
+	b := make([]byte, 126) // no 0, no 127, then we have 1-126 ASCII chars
+	for i := 0; i < 126; i++ {
+		b[i] = byte(i + 1)
+	}
```

Comment:
```suggestion
	// Fill the slice with ASCII characters including 1 up until 126.
	b := make([]byte, 126)
	for i := byte(0); i < 126; i++ {
		b[i] = i + 1
	}
```

Reply:
Well, it's better to have a comment about the change, I didn't know why it's necessary 😂

I guess the generated assemble code might be similar. They all need to do `MOV {target}, AL`, using a 8-bit register.